src/AppBundle/Entity/Job.php

```diff
@@ -484,6 +485,11 @@ public function prePersist()
     {
         $this->createdAt = new \DateTime();
         $this->updatedAt = new \DateTime();
+
+        if (!$this->getExpiresAt()) {
+            $now = $this->getCreatedAt() ? $this->getCreatedAt()->format('U') : time();
+            $this->expiresAt = new \DateTime('@'.$now + 86400 * 30);
```

Comment:
probably it's better to write:
```php
$this->expiresAt = (clone $this->createdAt)->modify('+30 days');
```
instead of 
```php
$now = $this->getCreatedAt() ? $this->getCreatedAt()->format('U') : time();
$this->expiresAt = new \DateTime('@'.$now + 86400 * 30);
```